### ISSUE TITLE
Remove tracing-gum

### ISSUE THREAD

**Issue by alexggh** [paritytech/polkadot-sdk#5881]
With jaeger tracing  removed in https://github.com/paritytech/polkadot-sdk/pull/5875, the reason for tracing-gum existence is gone, so we should consider removing it as well. 

See  https://github.com/paritytech/polkadot-sdk/pull/5875#pullrequestreview-2338025898 for more details.

**Comment by AndreiEres** [comment]
If `gum::warn_if_frequent!` is the only reason that holds it we can handle it in another place imho.

**Comment by programskillforverification** [comment]
I would like to work on this

**Comment by Krayt78** [comment]
Do you guys still want that done ?

**Comment by masonreid1729** [comment]
> Do you guys still want that done ?



Yes, I also received funds through this faucet, but unfortunately, I didn't find any more active wallets to claim according to their rules.

<img width="369" alt="Image" src="https://github.com/user-attachments/assets/bf9c7c3c-458c-4855-8fbf-32e8a4f04b23" />

**Comment by Captain1653** [comment]
Hi. 

Removed `tracing-gum` usage from [polkadot-node-core-approval-voting-parallel](https://github.com/paritytech/polkadot-sdk/pull/13270).

If this issue is still relevant, I can take care of the remaining pallets as well. I can process them in small, per-pallet PRs for easier review.
